### PR TITLE
feature: Protocol reward consumer

### PR DESCRIPTION
**Pull request by adamgobes**
This PR adds a consumer to hit the community NFT API on their first lend event and repayment event. The backed community NFT API then gives them 1 Activity XP

### CODE REVIEW

File: pages/api/events/consumers/communityNFTAwarder.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    } else if (eventName === 'RepaymentEvent') {
+      oldestEvent = await getOldestRepaymentEventForUser(
+        configs[network],
+        event.loan.borrowTicketHolder,
+      );
+    } else {
+      res.status(200);
+    }
+
+    console.log({ oldestEvent });
```
Comment (cnasc) [inline pages/api/events/consumers/communityNFTAwarder.ts:54]: nit: stray console

**Comment by cnasc** [comment]
might also be good to get some unit tests on these

File: pages/api/events/consumers/communityNFTAwarder.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        event.loan.borrowTicketHolder,
+      );
+      involvedAddress = oldestEvent?.repayer;
+    } else {
+      res.status(200).json('Community NFT Activity awarder successfully ran');
+      return;
+    }
+
+    if (!oldestEvent) {
+      // TODO(adamgobes): capture exception here
```
Comment (cnasc) [inline pages/api/events/consumers/communityNFTAwarder.ts:55]: @adamgobes  are we going to do this in this PR?

**Review by cnasc — APPROVED** [review]
One comment, otherwise LGTM

File: pages/api/events/consumers/communityNFTAwarder.ts (hunk 2 above)
Comment (adamgobes) [reply pages/api/events/consumers/communityNFTAwarder.ts:55]: nope will do in a follow up once i understand more closely when this would happen

**Pull request merged**